src/transformers/utils/quantization_config.py

```diff
@@ -590,6 +597,8 @@ def __init__(
         self.zero_point = zero_point
         self.version = version
         self.backend = backend
+        self.use_exllama = use_exllama
+        self.exllama_config = exllama_config
```

Comment:
I'm wondering if we could not just add exllama in the `version` attribute instead. We would remove `use_exllama` and `exllama_config`. `exllama_config` can be introduced in a later PR with we need to add exllama related kwargs. Is Exllama kernel compatible with GEMM and GEMV models ? Because I remember that you can't switch from GEMM to GEMV and vice versa.   cc @younesbelkada.

Reply:
That would be great indeed ! Let's raise an error with the `version` is "GEMV" and the user want to switch to `exllama` for now. As for putting "exllama" in `version`, I still think it will be better since we will have less args + easier to manage.

Reply:
done!